cfgov/cal/views.py

```diff
@@ -122,8 +122,10 @@ def display(request, pdf=False):
             'range_end':range_end,
         })
 
-        if pdf and PDFreactor:
-            return pdf_response(request, context)
+        if pdf:
+            template_name = 'about-us/the-bureau/leadership-calendar/print/index.html'
```

Comment:
The template_name is not used if PDFreactor is true, you can move this line below the condition if desired.